src/containers/Storage/Storage.tsx

```diff
@@ -107,6 +107,7 @@ export const Storage = ({database, viewContext, nodeId, groupId, pDiskId}: Stora
             groupId,
             pDiskId,
             shouldUseGroupsHandler: groupsHandlerAvailable,
+            fieldsRequired: 'all',
```

Comment:
It's possible to request data based on columns for not paginated storage. However, there is a problem: when you update columns, needed data fields are updated and data reloaded. For paginated table it isn't the issue, because we load only 50 entities, but for not paginated tables requests takes to long to load, so we need to avoid data reloads if possible